controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -557,6 +557,20 @@ void cli_bml::setFunctionsMapAndArray()
         " is-single-scan - 0 for continuous-scan results (default), 1 for single scan.",
         static_cast<pFunction>(&cli_bml::get_dcs_scan_results_caller), 2, 3, STRING_ARG, INT_ARG,
         INT_ARG);
+    insertCommandToMap("bml_client_get_client_list", "", "Get client list.",
+                       static_cast<pFunction>(&cli_bml::client_get_client_list_caller), 0, 0);
+    insertCommandToMap(
+        "bml_client_set_client",
+        "<sta_mac> <selected_bands> <stay_on_initial_radio> <stay_on_selected_device>",
+        "Set client with the given STA MAC:"
+        " selected_bands - 1 for 2.4G, 2 for 5G, 3 for both, 0 for Disabled"
```

Comment:
the description here should be aligned to the new implementation using the bitwise

Reply:
Updated the description